autopush/router/webpush.py

```diff
@@ -47,10 +47,11 @@ def _crypto_headers(self, notification):
             encoding=headers["content-encoding"],
             encryption=headers["encryption"],
         )
-        # AWS cannot store empty strings, so we only add the encryption-key if
-        # its present to avoid empty strings.
-        if "encryption-key" in headers:
-            data["encryption_key"] = headers["encryption-key"]
+        # AWS cannot store empty strings, so we only add these keys if
+        # they're present to avoid empty strings.
+        for name in ["encryption-key", "crypto-key"]:
+            if name in headers:
+                data[name] = headers[name]
```

Comment:
@tomoyukilabs You actually spotted a bug. As currently written, this will break data delivery for all clients. Thanks for catching that, and your comment on Bugzilla! :eyes: I'll send a PR soon.